client/src/client.js

```diff
@@ -2,8 +2,10 @@ import { ApolloClient, InMemoryCache, createHttpLink } from "@apollo/client";
 
 import { setContext } from "@apollo/client/link/context";
 
+const uri = process.env.REACT_APP_GRAPHQL_ENDPOINT ?? "http://localhost:4000/graphql";
```

Comment:
process env wont work while live.

Reply:
this will work , in heroku, open settings tab and then reveal env vars
https://www.youtube.com/watch?v=1G6e1cqqz3w
see the video referenced